### PR TITLE
Add a placeholer URDF for the AMP mount, update meshes

### PR DESCRIPTION
**Pull request by civerachb-cpr**
The AMP frame is undergoing some redesigns right now, so the final dimensions and STL are not available. But this adds a placeholder we can use for developing the software & doing validation of the generator until the final CAD files are available.

### CODE REVIEW

**Comment by civerachb-cpr** [comment]
A300 with placeholder AMP mount + sensors, treaded wheels, updated meshes.

Screenshot from 2024-12-02 14-22-10

File: clearpath_platform_description/urdf/a300/mobility/diff_drive.urdf.xacro [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
     <!-- Wheel Properties -->
-    <xacro:if value="${wheel == 'default' or wheel == 'smooth'}">
+    <xacro:if value="${wheel == 'smooth'}">
       <xacro:property name="wheel_length" value="0.1143"/>
       <xacro:property name="wheel_radius" value="0.163312"/>
       <xacro:property name="wheel_mesh" value="wheel_smooth.dae"/>
     </xacro:if>
 
-    <xacro:if value="${wheel == 'tread'}">
+    <xacro:if value="${wheel == 'default' or heel == 'tread'}">
```
Comment (luis-camero) [inline clearpath_platform_description/urdf/a300/mobility/diff_drive.urdf.xacro]: Typo: `heel` should be `wheel`

**Review by luis-camero — CHANGES_REQUESTED** [review]

File: clearpath_platform_description/urdf/a300/mobility/diff_drive.urdf.xacro (hunk 1 above)
Comment (civerachb-cpr) [reply clearpath_platform_description/urdf/a300/mobility/diff_drive.urdf.xacro]: Whoops. Fixed.

**Comment by civerachb-cpr** [comment]
Sorry about the commits & reversions; accidentally pushed some sim-related stuff I was working on to the wrong branch.

This should be ready for review & (hopefully) merging now.

**Review by luis-camero — APPROVED** [review]

**Comment by civerachb-cpr** [comment]
I'm just massaging this to get the Source CI to pass and then I'll merge it.

**Pull request merged**